indico/modules/events/registration/util.py

```diff
@@ -995,3 +997,22 @@ def get_persons(registrations, include_accompanying_persons=False):
                             'registration': registration,
                             'is_accompanying': True} for person in registration.accompanying_persons)
     return persons
+
+
+def resize_uploaded_registration_picture(source, size):
+    """Resize uploaded registration picture to the given size."""
+    try:
+        picture = Image.open(source)
+        picture = ImageOps.exif_transpose(picture)
+        size_x, size_y = picture.size
+        if max(size_x, size_y) < size:
+            source.seek(0)
+            return source
```

Comment:
Does the `picture` object need to be closed before returning here?

Reply:
I don't think we close the object from `Image.open()` anywhere else

Reply:
by `picture` you mean `source` right? If so then yes it should be closed first

Reply:
maybe i'm mistaken, but by close I assumed you meant returning to the beginning of the file with source.seek(0)?

Reply:
What I meant is calling `Image.close(picture)` but as Adrian pointed out we normally don't do it (even though in one place we do use `with Image.open(...)`)